### PR TITLE
Add AWS account-level tags documentation

### PR DESCRIPTION
**Pull request by biancaburtoiu**
This PR adds documentation on how AWS account-level tags are supported in Kubecost Assets view.

addresses https://github.com/kubecost/cost-analyzer-helm-chart/issues/946

### CODE REVIEW

File: aws-out-of-cluster.md [hunk 1]
```diff
@@ -42,6 +42,16 @@ In order to make the custom Kubecost AWS tags appear on the cost and usage repor
 
 Instructions for enabling user-defined cost allocation tags [here](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/activating-tags.html)
 
+## Viewing account-level tags
+
+You can view AWS account-level tags in Kubecost; tags are applied to all the resources defined under a given AWS account. You can filter AWS resources in the Kubecost Assets View (or API) by account-level tags by adding them ('tag:value') in the Label/Tag filter. 
+
+If a resource has a label with the same name as an account-level tag, the resource label value will take precedence; it won't be overriden by the value of the account-level tag.
```
Comment (nikovacevic) [inline aws-out-of-cluster.md:49]: "overriden" => "overridden"

**Review by nikovacevic — APPROVED** [review]
Just one typo, but otherwise looks good!

**Comment by dwbrown2** [comment]
Might be worth noting this feature is available in v1.93?

cc @kirbsauce as fyi

**Comment by kirbsauce** [comment]
does the feature also include GCP?

**Comment by dwbrown2** [comment]
Just looking at docs, this change does not appear to. Not sure if Bianca also implemented GCP or not.

**Comment by biancaburtoiu** [comment]
Apologies for the late reply, at the moment this is only valid for AWS! @kirbsauce @dwbrown2

**Comment by AdamStack18** [comment]
@dwbrown2 @kirbsauce Do we need to extend this to work with GCP?

**Comment by kirbsauce** [comment]
I think so, yes. The original request was actually for GCP and AWS was added later...then implemented first.

**Pull request merged**